src/algebra/big_operators.lean

```diff
@@ -965,6 +965,9 @@ calc (∑ x in s₁, f x) ≤ (∑ x in s₂ \ s₁, f x) + (∑ x in s₁, f x)
   ... = ∑ x in s₂ \ s₁ ∪ s₁, f x : (sum_union sdiff_disjoint).symm
   ... = (∑ x in s₂, f x)         : by rw [sdiff_union_of_subset h]
 
+lemma sum_mono_set_of_nonneg (hf : ∀ x, 0 ≤ f x) : monotone (λ s : finset α, s.sum f) :=
```

Comment:
Should this use the new `∑ x in s, f x` notation?

Reply:
Done.